src/client/Microsoft.Identity.Client/Internal/Requests/Interactive/AuthCodeRequestComponent.cs

```diff
@@ -193,8 +193,12 @@ private UriBuilder CreateInteractiveRequestParameters(IDictionary<string, string
                 //The CCS header is used by the CCS service to help route requests to resources in Azure during requests to speed up authentication.
                 //It consists of either the ObjectId.TenantId or the upn of the account signign in.
                 //See https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2525
```

Comment:
This comment should be on the main "GetCcsHeader()" method that was moved to the token client

Reply:
done. tx.